models/cart/src/shipping-info/transformers.ts

```diff
@@ -0,0 +1,30 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TShippingInfo, TShippingInfoGraphql } from './types';
+
+const buildFields: (keyof TShippingInfo)[] = [
+  'taxCategory',
+  'shippingMethod',
+  'price',
+  'shippingRate',
+  'taxedPrice',
+  'taxRate',
+  'deliveries',
+  'discountedPrice',
+];
+
+const transformers = {
+  default: Transformer<TShippingInfo, TShippingInfo>('default', {
+    buildFields,
+  }),
+  rest: Transformer<TShippingInfo, TShippingInfo>('rest', {
+    buildFields,
+  }),
+  graphql: Transformer<TShippingInfo, TShippingInfoGraphql>('graphql', {
+    buildFields,
+    addFields: () => ({
```

Comment:
We will need to use the `replaceFields` prop to handle the difference in values for some of the types and also adding the ones which are specific to the GraphQL representation.

As an example, `taxCategory` is of type `TaxCategoryReference` in the REST representation but it is a `TTaxCategoryGraphql` type for the GraphQL. Also, in GraphQL we have the `taxCategoryRef` extra property of type `TReferenceGraphql`.

You can take a look at the [ProductProjection transformers](https://github.com/commercetools/test-data/blob/main/models/product-projection/src/transformers.ts#L95) to get an example of how to manage this.